### PR TITLE
Improve calculator plugin

### LINKED ISSUE #99: Feature: Improve calculator plugin
I was planning to add a couple of improvements to the calculator plugin.

- New setting to set the output decimal separator to either dot or comma. Currently, it is fixed to a dot. The current locale can be used as "default" here.
- Enable calculation with dot and comma. Currently, only dots are allowed as input value.
- Add new settings to set max calculation precision. 
- Solve rounding errors in a couple of circumstances. (Wox-launcher#2349)

This should also close issue Wox-launcher#2106

### PR DESCRIPTION
**Pull request by SysC0mp**
Closes #99 

- Add new setting to choose the output separator (Closes https://github.com/Wox-launcher/Wox/issues/2106)
- Add new setting to set the maximum decimal places for the result (should help with https://github.com/Wox-launcher/Wox/issues/2349)
- Input can use either , or . as decimal separator

### CODE REVIEW

**Comment by jjw24** [comment]
Resolves https://github.com/Wox-launcher/Wox/pull/2724 and https://github.com/Wox-launcher/Wox/issues/2720

**Comment by jjw24** [comment]
@SysC0mp looking really good. When the PR is ready, please select a reviewer.

**Comment by SysC0mp** [comment]
@jjw24 I think the PR should be ready for review. 👍

File: Wox.Infrastructure/UI/EnumBindingSource.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Windows.Markup;
+
+namespace Wox.Infrastructure.UI
+{
+    public class EnumBindingSourceExtension : MarkupExtension
```
Comment (jjw24) [inline Wox.Infrastructure/UI/EnumBindingSource.cs]: @SysC0mp is this class used?

**Comment by jjw24** [comment]
Hey @SysC0mp not sure how to work this, it doesnt seem to like me:

image

image

**Comment by SysC0mp** [comment]
Hi, you can only use decimal separators atm, no thousands separator. As far as I remember, that was not possible before also, but it could be a good additional improvement to add that.

**Comment by jjw24** [comment]
Yes of course, lol, I cant read today. Awesome PR, thank you for your contribution.

**Review by jjw24 — APPROVED** [review]

**Pull request merged**